base/lthooks.dtx

```diff
@@ -321,14 +321,17 @@
 %    When the optional argument \meta{label} is not provided, the
 %    \meta{default label} is used (see section~\ref{sec:default-label}).
 %
-%    If the code for that \meta{label} wasn't yet added to the
-%    \meta{hook}, an order is set so that when some code attempts to add
-%    that label, the removal order takes action and the code is not
-%    added.
+%    If there is no code under the \meta{label} in the \meta{hook},
+%    or if the \meta{hook} does not exist, an error is raised when
```

Comment:
I wonder if this should be an error or just a warning (or not even that). I would probaby go for a warning only even with your explanation below

Reply:
I have mixed feelings.  On the one hand, an error sounds more discouraging if someone wants to use `\RemoveFromHook` and is faced with the trouble of finding out which labels are there and which are not.  On the other hand, trying to remove a label that doesn't exist and then end up doing nothing is not really serious, so maybe not worth of an error...

I suppose I'll change to a warning then...